linux/inotify.c

```diff
@@ -13,7 +13,7 @@ int main(int argc, char *argv[])
         goto err;
     }
 
-    int ret = inotify_add_watch(ifd, "./", IN_MOVED_TO | IN_MOVED_FROM);
+    int ret = inotify_add_watch(ifd, "./", IN_ALL_EVENTS);

```

Comment:
We will get what the originally wanted for all events, please refer to https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/inotify.h#L66.